ground/src/main/java/com/google/android/ground/ui/offlineareas/OfflineAreaListItem.kt

```diff
@@ -99,6 +105,16 @@ fun OfflineAreaListItem(
   }
 }
 
+@Composable
+private fun OfflinePinImage() {
+  Icon(
+    imageVector = ImageVector.vectorResource(id = R.drawable.ic_offline_pin),
+    contentDescription = stringResource(id = R.string.offline_area_list_item_icon),
+    tint = MaterialTheme.colorScheme.primary,
+    modifier = Modifier.size(dimensionResource(R.dimen.spacing_xlarge)),
```

Comment:
Using spacing dimen for size seems incorrect. Can you create new dimens for size?

Reply:
the plan was to use the generic values wherever required like 8dp 4dp etc

are you suggesting duplicating the values with _size suffix?